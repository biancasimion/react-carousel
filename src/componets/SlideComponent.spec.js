import React from 'react';
import Slide from './SlideComponent';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer
    .create(<Slide image={
      {
        url: 'test.com',
        index: 0,
        tags: 'test test',
      }

    }/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});