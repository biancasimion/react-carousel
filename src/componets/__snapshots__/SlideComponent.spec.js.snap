// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="slide"
>
  <img
    alt=""
    src="test.com"
  />
  <h4>
    test test
  </h4>
</div>
`;
